### ISSUE TITLE
1.The dataset download link is google.com;

### ISSUE THREAD

**Issue by windelvin** [ringochuchudull/Event-Intensity-Stereo#1]

**Comment by windelvin** [comment]
and when I try to run run_inference.sh, it shows "FileNotFoundError: [Errno 2] No such file or directory: '/home/ma-user/work/Event-Intensity-Stereo-main/./pretrained_models/model_weight.ckpt'"

**Comment by XMU-qcj** [comment]
Yes, the link is faulty, can you please change it to the correct link, thanks for your work!
